django/cantusdb_project/cantusdb/settings.py

```diff
@@ -164,3 +166,9 @@
 DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
 
 EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+
+if DEBUG:
```

Comment:
What is this doing?

Reply:
I followed the instructions from https://django-debug-toolbar.readthedocs.io/en/latest/installation.html where it says this code is needed to configure INTERNAL_IPS correctly with Docker. Not sure exactly what's going on though. I just tested it without this and it still works so I'll probably end up removing it.

Reply:
Makes sense!